tests/sentry/tasks/test_appconnect.py

```diff
@@ -31,6 +32,16 @@ def config(self):
             orgName="Example Com",
         )
 
+    @pytest.fixture
+    def build(self):
+        return appconnect.BuildInfo(
+            app_id="123",
+            platform="iOS",
+            version="0.0.1",
+            build_number="001",
```

Comment:
I don't think they use trailing zeros, but it shouldn't matter much as we (should) treat this as an opaque string anyway

Reply:
Swapped these to slightly less obvious placeholder values; It looks like some devs use dates as build numbers, and my personal experience matches this so I stuffed one into the build number field as a loose check to make sure those work.